k8s/application/server/deployment.yaml

```diff
@@ -52,6 +61,7 @@ spec:
         resources:
           # Autopilot only considers requests (see https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-resource-requests#resource-limits)
           requests:
-            cpu: 250m
-            memory: 512Mi
+            # Experiment: using the minimum pod resources https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-resource-requests#compute-class-min-max
+            cpu: 150m # + `isito-proxy` container = 250m
```

Comment:
I did try setting these values to something less than the specified ones but autopilot would just bump it up during `kubectl create`. In this case, however, explicitly setting `istio-proxy` resource configs might help.